src/docs/sdk/event-payloads/properties/event_id.mdx

```diff
@@ -0,0 +1,10 @@
+`event_id`:
+
+: _Required_. Hexadecimal string representing a uuid4 value. The length is exactly 32
```

Comment:
Just because I'm curious, any reason we opted to use uuid? It might be faster on some platforms to just generate a random `byte` array and encode to string. Ditto with `trace_id` and `span_id`